api_tests/tests/herc20_hbit_respawn.ts

```diff
@@ -121,7 +121,7 @@ describe("herc20-hbit-respawn", () => {
     );
 
     it(
-        "hbit-herc20-bob-down-misses-alice-redeem",
+        "hbit-herc20-bob-down-misses-alice-redeems",
```

Comment:
I always read this as "bob down, misses alice's redeem" in which case the added `s` doesn't make sense but I guess one can argue both ways :)

Reply:
I was wrong, I incorrectly inferred that there was non-uniformity. Have added a patch to use singular as suggested.